### PR TITLE
change healthy response so it can pass aws load balance health check

### PR DESCRIPTION
**Pull request by rongshen-daqri**
@unixorn 

I have never used go before, not sure if I'm writing the right code here. What I'm trying to do is adding an option for passing any response you like, and set the default one to be '200 OK' so aws load balance health check can recognize it.

### CODE REVIEW

File: apgar-probe.go [hunk 1]
```diff
@@ -155,6 +156,7 @@ func main() {
 	flag.StringVar(&documentRoot, "document-root", "/var/lib/apgar", "Document root")
 	flag.StringVar(&healthCheckName, "healthcheck-name", "healthCheck", "health check script suffix")
 	flag.StringVar(&healthCheckTree, "healthcheck-tree", "/var/lib/apgar", "Directory tree to search for health checks")
+	flag.StringVar(&healthyResponse, "healthy-response", "200 OK\n", "The response string if status is healthy")
```
Comment (rongshen-daqri) [inline apgar-probe.go:159]: Do you think we need the new line here? aws document just said "200 OK", not sure if new line will affect.

File: apgar_tests.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def test_multiple_passing
     run = `./apgar-probe --document-root tmp --healthcheck-tree fixtures/003-multiple-passing`
```
Comment (rongshen-daqri) [inline apgar_tests.rb:33]: Is there an instruction on how to run tests? I tried just run "bundle install; ruby apgar_test.rb" but I got failures on `Errno::ENOENT: No such file or directory - ./apgar-probe`

**Comment by rongshen-daqri** [comment]
I just realized this is may not be the correct way to make load balancer work with apgar. As long as load balancer can get response from /status page, it will think the instance is healthy. However, status page is always available, only the response is different. Close the PR.

File: apgar_tests.rb (hunk 2 above)
Comment (unixorn) [reply apgar_tests.rb:33]: Sorry, do `rake test`

**Pull request closed without merging**